Spec/RealtimeClientPresence.swift

```diff
@@ -77,6 +77,35 @@ class RealtimeClientPresence: QuickSpec {
                 }
             }
 
+            // RTP4
+            it("should receive all 250 members") {
+                let options = AblyTests.commonAppSetup()
+                var clientSource: ARTRealtime!
+                defer { clientSource.close() }
+
+                let clientTarget = ARTRealtime(options: options)
+                defer { clientTarget.close() }
+                let channel = clientTarget.channels.get("test")
+
+                channel.presence.subscribe { member in
+                    expect(member.action).to(equal(ARTPresenceAction.Enter))
```

Comment:
Test is failing 😫. The client is broken. I will change to pending after rebasing with master.